src/command_modules/azure-cli-cdn/azure/cli/command_modules/cdn/_help.py

```diff
@@ -0,0 +1,160 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-import
+from azure.cli.core.help_files import helps
+
+# pylint: disable=line-too-long
+helps['cdn'] = """
+    type: group
+    short-summary: Manage Azure Content Delivery Networks (CDN)
+"""
+
+helps['cdn profile create'] = """
+    type: command
+    examples:
+        - name: Create a CDN profile using Verizon premium CDN
+          text: >
+            az cdn profile create -g group -n profile --sku Premium_Verizon
```

Comment:
Should the examples use <group>  and <profile>?  Otherwise the implication is those values should work.  If they do, then that's fine.